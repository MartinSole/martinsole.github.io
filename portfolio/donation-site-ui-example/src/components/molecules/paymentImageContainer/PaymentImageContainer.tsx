import { Image } from '../../atoms'

const PaymentImageContainer = () => {
  return (
    <footer className='logoContainer'>
      <Image src="src/images/logos/Fundraising Regulator.svg" alt="Funraising regulator" />
      <Image src="src/images/logos/Visa.svg" alt="Visa" />
      <Image src="src/images/logos/Mastercard.svg" alt="Mastercard" />
      <Image src="src/images/logos/Apple Pay.svg" alt="Apple pay" />
      <Image src="src/images/logos/Google Pay.svg" alt="Google Pay" />
      <Image src="src/images/logos/Paypal.svg" alt="Paypal" />
    </footer>
  )
}

export default PaymentImageContainer