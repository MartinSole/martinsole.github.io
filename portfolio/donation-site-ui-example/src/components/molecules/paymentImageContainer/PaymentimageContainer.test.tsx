import '@testing-library/jest-dom'
import { render } from "@testing-library/react"
import PaymentImageContainer from "./PaymentImageContainer"

test("Renders the Text component", () => {
   const {container} = render(<PaymentImageContainer />)
    expect(container).toMatchSnapshot()
})
