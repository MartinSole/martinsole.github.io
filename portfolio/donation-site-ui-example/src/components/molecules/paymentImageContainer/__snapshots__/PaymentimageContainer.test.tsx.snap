// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Text component 1`] = `
<div>
  <footer
    class="logoContainer"
  >
    <img
      alt="Funraising regulator"
      src="src/images/logos/Fundraising Regulator.svg"
    />
    <img
      alt="Visa"
      src="src/images/logos/Visa.svg"
    />
    <img
      alt="Mastercard"
      src="src/images/logos/Mastercard.svg"
    />
    <img
      alt="Apple pay"
      src="src/images/logos/Apple Pay.svg"
    />
    <img
      alt="Google Pay"
      src="src/images/logos/Google Pay.svg"
    />
    <img
      alt="Paypal"
      src="src/images/logos/Paypal.svg"
    />
  </footer>
</div>
`;
